src/test/java/api/support/builders/ItemBuilder.java

```diff
@@ -282,6 +324,30 @@ public ItemBuilder withCopyNumbers(List<String> copyNumbers) {
       this.permanentLoanTypeId,
       this.temporaryLoanTypeId,
       this.enumeration,
-      copyNumbers);
+      copyNumbers,
+      this.itemLevelCallNumber,
+      this.itemLevelCallNumberPrefix,
+      this.itemLevelCallNumberSuffix);
+  }
+
+  public ItemBuilder withCallNumber(
+    String itemLevelCallNumber,
+    String itemLevelCallNumberPrefix,
+    String itemLevelCallNumberSuffix) {
+    return new ItemBuilder(
```

Comment:
Personally, I think a blank line between the method definition and the implementation helps with readability here

Reply:
fix